src/Dialect/ONNX/ONNXOps.cpp

```diff
@@ -3424,6 +3452,35 @@ LogicalResult ONNXDropoutOp::inferShapes(
 //===----------------------------------------------------------------------===//
 // OneHotEncoder
 //===----------------------------------------------------------------------===//
+LogicalResult ONNXOneHotEncoderOp::verify() {
```

Comment:
Nit: the `verify` method for the other operations comes after the `inferShapes` method.